packages/firebase_crashlytics/test/firebase_crashlytics_test.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$Crashlytics', () {
+    const MethodChannel channel =
```

Comment:
For other plugins I've making the channel be `@visibleForTesting` and then just using it here. This is just to avoid unnecessary duplication as we need the channel to intercept method calls. I don't really feel strongly about it though.

Reply:
Done